cernopendata/modules/theme/bundles.py

```diff
@@ -29,6 +29,11 @@
 from flask_assets import Bundle
 from invenio_assets import NpmBundle
 
+from invenio_search_ui.bundles import js as _js
+
+# Override d3 version from search-ui
+_js.npm['d3'] = '~3.3.13'
```

Comment:
`npm install d3@3.3.13` should probably be added to `scripts/create-instance.sh` then.